common/character_interactions/00_blackmail_interactions.txt

```diff
@@ -12,7 +12,8 @@
 	is_shown = {
```

Comment:
There's a lot of errors mentioning our values and triggers. They can be found by searching `age_` in `error.log`.